twake/backend/node/src/services/online/service/index.ts

```diff
@@ -48,22 +45,42 @@ export default class OnlineServiceImpl implements TwakeServiceProvider, OnlineSe
           ack({ data: await this.getOnlineStatuses(request.data) });
         },
       );
+
+      event.socket.on(
+        "online:set",
+        async (request: OnlineGetRequest, ack: (response: OnlineGetResponse) => void) => {
+          this.logger.debug(`Got an online:set request for ${(request.data || []).length} users`);
+
```

Comment:
What you need to do here is more that:
```
 this.pubsubService.broadcastOnline([[event.user.id, false]]);
 this.setLastSeenOnline([event.user.id], Date.now(), false);
```

Reply:
Also you don't need to pass data in the online:set event, you can use the event.user.id from line 33.